code/modules/clothing/suits/marine_armor.dm

```diff
@@ -132,12 +132,6 @@
 	max_storage_space = 8 // Rifle, stock, extended barrel, grip/bipod.
 	max_w_class = 3 //Can fit larger items
 
-/obj/item/clothing/suit/storage/marine/M3E
```

Comment:
Keep the object, remove from vendors only.

Reply:
literally why its unbalanced as fuck

Reply:
For later use, events and whatnot. It has sprites and is fully functional.
Remove it from the marine vendors, just not from the game.